File: src/main/java/com/kingmang/lazurite/parser/pars/FunctionAdder.java
package com.kingmang.lazurite.parser.pars;

import com.kingmang.lazurite.parser.ast.FunctionDefineStatement;
import com.kingmang.lazurite.parser.pars.AbstractVisitor;


public final class FunctionAdder extends AbstractVisitor {

    @Override
    public void visit(FunctionDefineStatement s) {
        super.visit(s);
        s.execute();
    }
}

File: src/main/java/com/kingmang/lazurite/patterns/visitor/FunctionAdder.kt
package com.kingmang.lazurite.patterns.visitor;

import com.kingmang.lazurite.parser.AST.Statements.FunctionDefineStatement;


public final class FunctionAdder extends AbstractVisitor {

    @Override
    public void visit(FunctionDefineStatement s) {
        super.visit(s);
        s.execute();
    }
}
